### ISSUE TITLE
Some git-related glyphs seem slightly out of place

### ISSUE THREAD

**Issue by theckman** [chrissimpkins/Hack#30]
I'm coming from Menlo for Powerline while using zsh+powerline on OS X. When working in a git repository the glyphs seem a bit off to me.

Hack:
![hack](https://cloud.githubusercontent.com/assets/787332/9565869/2db164ec-4e9f-11e5-8790-3e3fc7175ae3.jpg)

Menlo:
![menlo](https://cloud.githubusercontent.com/assets/787332/9565872/3b285090-4e9f-11e5-9404-adcc11723b7c.jpg)

I feel like the arrow, as you've changed it, would feel more natural with the glyphs in Menlo.

**Comment by chrissimpkins** [comment]
The check mark and up arrow are not in the Powerline glyph set (i.e. those mapped into the Unicode private use area), those are coming from the regular set.  Let me leave this open for a bit to see if I can get additional feedback about the arrows for other uses.  We may be able to strike a balance between the arrow in Hack and that in Menlo.  IMO, the strokes in the Menlo arrow are far too large for the overall set (though it looks really nice when used here in Powerline!) and would be more appropriate for an extra bold / ultra weight.  Must admit, I haven't given these characters much love to this point because the focus has been on the source and not on displays/UI.  

Also, if you are interested/willing to help with Powerline testing, there is an open issue to address the alignment of the Powerline glyphs (#33).  Interestingly, your system seems to have the horizontal but not the vertical alignment issues that were reported there (seen in the left arrow heads on the rectangular colored boxes).

**Comment by chrissimpkins** [comment]
New test build with the first iteration of Powerline glyph alignment changes is now available.  See #33

**Comment by alerque** [comment]
I agree that part of the problem here is that the Menlo glyphs are actually out of sync with the font. They work okay for this usage in Powerline but they are not a good match for the normal meaning of the glyphs compared to the rest of the font.

On the other hand the current Hack glyphs seem to be a bit too far the other way. I would expect the checkmark and arrow glyphs to carry roughly the same size impressions. I actually think the somewhat exaggerated size of the checkbox is appropriate for a monospace font like this, but other glyphs such as arrows probably need to grow a a little bit to match.

One mistake on the part of most Powerline implementations is relying on a mix and match set of normal glyphs vs. private use glyphs. That is almost guaranteed to cause imbalances when used in some fonts. It isn't necessarily the job of any particular font to balance these concerns, but it would be nice to see Hack improve the balance of the glyphs that are coming out of the normal space. And since it's including the Powerline private space stuff as well, those elements can mostly be adjusted to match the built in glyphs too (excluding of course the full-frame expectations that some of the glyphs will touch the edge of the bounding box, see also [my  comment on issue 33](https://github.com/chrissimpkins/Hack/issues/33#issuecomment-136593699)).

**Comment by aclissold** [comment]
Another use case for these symbols is [zsh-git-prompt](https://github.com/olivierverdier/zsh-git-prompt), which I use religiously. In fact, it's probably the main reasons I've never been able to switch away from Menlo, because it looks weird in literally every other font besides Menlo and DejaVu.

**Menlo:**
<img width="388" alt="screen shot 2015-09-01 at 10 27 18 am" src="https://cloud.githubusercontent.com/assets/4397642/9607023/c6219412-5094-11e5-93ba-3d06b1176559.png">

**DejaVu Sans Mono:**
<img width="388" alt="screen shot 2015-09-01 at 10 27 33 am" src="https://cloud.githubusercontent.com/assets/4397642/9607027/c9e34ee2-5094-11e5-855c-a93c477f8476.png">

**Hack:**
<img width="388" alt="screen shot 2015-09-01 at 10 27 48 am" src="https://cloud.githubusercontent.com/assets/4397642/9607033/cef96074-5094-11e5-9584-40af4537d369.png">
